src/applications/disability-benefits/2346/components/ReviewPageBatteries.jsx

```diff
@@ -32,14 +32,14 @@ const ReviewPageBatteries = ({ selectedProductInfo, batterySupplies }) => (
 
 const mapStateToProps = state => {
   const supplies = state.form?.loadedData?.formData?.supplies;
-  const batterySupplies = supplies.filter(supply =>
-    supply.productGroup.includes('batteries'),
-  );
+  const batterySupplies =
+    supplies.length &&
+    supplies.filter(supply => supply.productGroup.includes('batteries'));
```

Comment:
This could be rewritten using optional chaining:

`supplies.length && supplies.filter(...)` becomes `supplies?.filter(...)`